src/providers/tokens.provider.ts

```diff
@@ -95,15 +105,12 @@ export default {
       injectedTokens: {
         [networkConfig.nativeAsset.address]: nativeAsset
       },
-      allowanceContracts: [
+      allowanceContracts: compact([
         networkConfig.addresses.vault,
-        ...(networkConfig.addresses.exchangeProxy
-          ? [networkConfig.addresses.exchangeProxy]
-          : []),
-        ...(GP_ALLOWANCE_MANAGER_CONTRACT_ADDRESS
-          ? [GP_ALLOWANCE_MANAGER_CONTRACT_ADDRESS]
-          : [])
-      ]
+        networkConfig.addresses.wstETH,
+        networkConfig.addresses.exchangeProxy,
+        GP_ALLOWANCE_MANAGER_CONTRACT_ADDRESS
```

Comment:
@garethfuller I assume you wanted to remove these addresses if they are "empty" right?
If so, lodash `compact` comes in handy for these types of situations.